src/components/layout/Footer/Footer.module.scss

```diff
@@ -85,13 +85,18 @@
     }
   }
   .emptySpace {
+    color: transparent;
```

Comment:
Chodziło mi o kolor font'u z transparentnego na ten pasujący, a nie tła ;) Bo teraz jak by klient wziął projekt i wpisał coś w tego div'a to będzie się zastanawiał czemu mu nie działa a ma tekst zlewający się z tłem ;)

Reply:
Tak teraz myslę, @Pawelsmolarski95  czemu nie użyjesz w footerze kolumn boostrapa czyli `col-` nie musiał byś tego sam pisać i kombinować z `emptySpaces`

Reply:
w sumie tak, usunąłem emptySpaces i zamiast tego użyłem bootstrapa i kolor jest już ogarnięty tak że jak ktoś faktycznie będzie cos wpisywał w to przygotowane pole to będzie mu się wyświetlało tak jak powinno w kolorach  tego dolnego footera